Python/PythonWithCpp/Ctypes/MathCalc/CMakeLists.txt

```
@@ -0,0 +1,20 @@
+cmake_minimum_required(VERSION 3.12.0)
+project(MathCalc)
+
+set(CMAKE_CXX_STANDARD 11)
```

```suggestion
target_compile_features(${CMAKE_PROJECT_NAME} PRIVATE cxx_std_11)
```
This is a more correct way to set desired c++ standard.

---

Only problem is that if I would commit this suggestion directly it would cause a build error - need to be much lower when target to compile is known.